tests/api_test/api_test.cpp

```diff
@@ -1001,8 +1001,6 @@ test_sock_addr_native_program_load_attach(const char* file_name)
         test_sock_addr_native_program_load_attach((const char*)"cgroup_sock_addr2_"##version ".sys"); \
     }
 
-DECLARE_REGRESSION_TEST_CASE("0.9.0")
```

Comment:
Maybe comment it out with a TODO and an issue number to put it back in here in the next release?

Reply:
Added.